src/main/java/org/myrobotlab/service/InMoovArm.java

```diff
@@ -203,7 +204,7 @@ public Servo getRotate() {
   }
 
   public String getScript(String inMoovServiceName) {
-    return String.format("%s.moveArm(\"%s\",%d,%d,%d,%d)\n", inMoovServiceName, side, bicep.getPos(), rotate.getPos(), shoulder.getPos(), omoplate.getPos());
+    return String.format((Locale)null,"%s.moveArm(\"%s\",%.2f,%.2f,%.2f,%.2f)\n", inMoovServiceName, side, bicep.getPos(), rotate.getPos(), shoulder.getPos(), omoplate.getPos());

```

Comment:
casting a null value to a locale object here doesn't make any sense.  Also, why are we limiting the resolution to 2 decimal places for the move command?

Reply:
"(Locale)null,"  should be removed.  If you are passing a locale in there, that's fine. but you're not.  It's just being passed as null.

Reply:
You should pass something like  Runtime.getLocale() instead of null.

Reply:
welcome Locale.English